puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/release/foreman-x-develop-release.groovy

```diff
@@ -57,7 +57,7 @@ pipeline {
                                 action: 'nightly',
                                 packages: obal_package_name ?: project_name,
                                 extraVars: [
-                                    'releasers': [ 'koji-foreman' ],
+                                    'releasers': releasers ?: [ 'koji-foreman' ],
```

Comment:
This broke hammer-cli-foreman-master: `groovy.lang.MissingPropertyException: No such property: releasers for class: WorkflowScript`